test/call.spec.js

```diff
@@ -131,14 +133,14 @@ describe("Test 'call' command", () => {
 			options: {},
 			actionName: "math.add",
 			jsonParams:
-				'{"a": 5, "b": "Bob", "c": true, "d": false, "e": { "f": "hello" } }',
-			meta: '{"meta_a": 5, "meta_b": "Bob", "meta_c": true, "meta_d": false, "meta_e": { "meta_f": "hello" } }',
-			rawCommand: `call math.add {"a": 5, "b": "Bob", "c": true, "d": false, "e": { "f": "hello" } } {"meta_a": 5, "meta_b": "Bob", "meta_c": true, "meta_d": false, "meta_e": { "meta_f": "hello" } }`,
+				"{\"a\": 5, \"b\": \"Bob\", \"c\": true, \"d\": false, \"e\": { \"f\": \"hello\" } }",
```

Comment:
Please revert these reformatting changes because it makes less readable the codes.

Reply:
Done.